auto_submit/lib/service/revert_review_template.dart

```diff
@@ -34,8 +36,7 @@ revert request listed above.
 
 Please do the following so that we may track this issue to completion:
 1. Add the reviewer of revert pull request as the assignee of this issue.
-2. Add the label 'revert_review' to this issue.
```

Comment:
This sounds likes something the bot should do instead of the human.

Reply:
That is an excellent point! It should just be added right when the issue is created.